tests/features/collection/homepage.feature

```diff
@@ -6,11 +6,11 @@ Feature: Collection homepage
 
   Background:
     Given users:
-      | Username |
-      | Frodo    |
-      | Boromir  |
-      | Legoloas |
-      | Gimli    |
+      | Username | Status |
+      | Frodo    | 1      |
```

Comment:
As @pfrenssen suggested, let's take the chance now to replace 1 with `active` and 0 with `blocked` to give business meaning to those numbers.